waspc/test/AnalyzerTest.hs

```diff
@@ -239,10 +240,11 @@ spec_Analyzer = do
                                    intercalate
                                      "\n"
                                      [ "Type error:",
-                                       "  For dictionary field 'dependencies':",
-                                       "    For list element:",
-                                       "      Expected type: (string, string)",
-                                       "      Actual type:   (string, number)"
+                                       "  Expected type: (string, string)",
+                                       "  Actual type:   (string, number)",
+                                       "",
+                                       "  -> For dictionary field 'dependencies':",
```

Comment:
Looking at this, would it make sense to further indent context error messages, to make it clear they are additional information?

Smth like
```
                                     [ "Type error:",
                                       "  Expected type: (string, string)",
                                       "  Actual type:   (string, number)",
                                       "",
                                       "    -> For dictionary field 'dependencies':",
                                       "      -> In list element"
```

Reply:
I prefer it without the further indent, but can't really explain why. They're already indented under `Type Error`, and the empty line does a good-enough job of separating the additional info, I think. Indenting the first context message looks a bit weird to me.

I'd personally leave it like this., but I don't have a strong preference. I can change it if you do.

Reply:
Ok, no worries, let's leave it as it is then!